packages/transaction-controller/src/types.ts

```diff
@@ -167,6 +167,16 @@ type TransactionMetaBase = {
    */
   firstRetryBlockNumber?: string;
 
+  /**
+   * Whether the transaction is the first time interaction.
+   */
+  isFirstTimeInteraction?: boolean;
+
+  /**
+   * Whether the first time interaction is disabled.
+   */
+  isFirstTimeInteractionDisabled?: boolean;
```

Comment:
Why do we need this property, can we not infer this from `isFirstTimeInteraction` being `undefined` and the preference in the client?

Reply:
I also thought this and maybe it would be cleaner to have specific property. Since you also noticed that - I removed `disabled` property